lib/routes/tencent/wechat/ce.js

```diff
@@ -1,33 +1,62 @@
 const parser = require('@/utils/rss-parser');
 const got = require('@/utils/got');
 const cheerio = require('cheerio');
+const { fixArticleContent } = require('@/utils/wechat-mp');
+const { parseDate } = require('@/utils/parse-date');
+
+// any UA containing "RSS" can pass the check
```

Comment:
There's not really true. I'm still seeing cf challenges when testing with both US GCP or my home IP.

Reply:
Well, both of my VPSes, one in HK while one in SG, and my PC (using CERNET) can pass the check easily...

```sh
> curl https://posts.careerengine.us/author/5b0d5cbbd65e20012e6d8307/rss -A "Mozilla/5.0 (X11; Linux x86_64) RSS Reader" -I
HTTP/2 200
...
```

Reply:
It seems CF doesn't like me.
```
$ curl api.ip.sb -4
219.78.x.x
$ curl https://posts.careerengine.us/author/5b0d5cbbd65e20012e6d8307/rss -A "Mozilla/5.0 (X11; Linux x86_64) RSS Reader" -I -4
HTTP/2 403
date: Fri, 08 Apr 2022 11:05:51 GMT
content-type: text/html; charset=UTF-8
cf-chl-bypass: 1
...
```